kt-extensions/src/main/java/com/mindera/skeletoid/kt/extensions/utils/ListExt.kt

```diff
@@ -0,0 +1,3 @@
+package com.mindera.skeletoid.kt.extensions.utils
+
+fun <T> List<T>.nullIfEmpty() = if (isEmpty()) null else this
```

Comment:
Don't we have the same function in the file:

`kt-extensions/src/main/java/com/mindera/skeletoid/kt/extensions/utils/List.kt`

Reply:
I've deleted that file in a previous PR

Reply:
Oh, nvm. This was a rename